src/utils/__tests__/entitySpec.test.unit.js

```diff
@@ -86,4 +86,14 @@ describe('EntitySpec', () => {
     const result = EntitySpec.fromUrl('https://rubygems.org/gems/tzinfo/versions/1.2.5')
     expect(result.toString()).toEqual('gem/rubygems/-/tzinfo/1.2.5')
   })
+
+  it('returns a debian path with a version', async () => {
+    const result = EntitySpec.fromUrl('https://sources.debian.org/api/src/amoeba/1.1-30/')
+    expect(result.toString()).toEqual('deb/debian/-/amoeba/1.1-30')
```

Comment:
For Debian packages there is going to be a namespace, for example deb/debian/amd64/amoeba/1.1-30.

Reply:
Do you have some rules that i can use to build a proper parser?
For example, using the URL above, how do you know that the namespace should be amd64 ?

Reply:
The namespace is the architecture. As with other name-spaced packages, the users should be able to provide it. Does it make sense?

Reply:
@geneh not totally. What does should change from a curator point of view? Should the user really know if he's curating a package on a specific architecture?
Probably i'm missing something.
@iamwillbar thoughts?